### PR TITLE
Drop nxp89xx

### PR DESCRIPTION
**Pull request by quaresmajose**
All of this is now been available upstream
https://github.com/Freescale/meta-freescale/pull/1346

### CODE REVIEW

File: meta-lmp-bsp/recipes-kernel/linux-firmware/linux-firmware_%.bbappend [hunk 1]
```diff
@@ -39,20 +36,6 @@ do_install:append:beaglebone-yocto() {
...
-    install -m 0644 ${WORKDIR}/imx-firmware/nxp/FwImage_8987/txpwrlimit_cfg_8987.conf  ${D}${nonarch_base_libdir}/firmware/nxp
-
-    # Install NXP Connectivity 8997PCIE firmware
-    install -m 0644 ${WORKDIR}/imx-firmware/nxp/FwImage_8997/ed_mac_ctrl_V3_8997.conf  ${D}${nonarch_base_libdir}/firmware/nxp
-    install -m 0644 ${WORKDIR}/imx-firmware/nxp/FwImage_8997/pcieuart8997_combo_v4.bin ${D}${nonarch_base_libdir}/firmware/nxp
-    install -m 0644 ${WORKDIR}/imx-firmware/nxp/FwImage_8997/txpwrlimit_cfg_8997.conf  ${D}${nonarch_base_libdir}/firmware/nxp
-
     # Install Murata 1MW NVRAM and HCD files
     install -m 0644 ${WORKDIR}/imx-firmware/cyw-wifi-bt/1MW_CYW43455/BCM4345C0.1MW.hcd ${D}${nonarch_base_libdir}/firmware/brcm/BCM4345C0.hcd
     install -m 0644 ${WORKDIR}/imx-firmware/cyw-wifi-bt/1MW_CYW43455/brcmfmac43455-sdio.txt ${D}${nonarch_base_libdir}/firmware/brcm/brcmfmac43455-sdio.fsl,${MACHINE}.txt
```
Comment (ricardosalveti) [inline meta-lmp-bsp/recipes-kernel/linux-firmware/linux-firmware_%.bbappend:41]: We should upstream this piece and drop it later as well.

**Review by ricardosalveti — APPROVED** [review]
LGTM

File: meta-lmp-bsp/recipes-kernel/linux-firmware/linux-firmware_%.bbappend (hunk 1 above)
Comment (quaresmajose) [reply meta-lmp-bsp/recipes-kernel/linux-firmware/linux-firmware_%.bbappend:41]: I will take care of that.

**Review by MrCry0 — APPROVED** [review]
LGTM

**Comment by quaresmajose** [comment]
tested locally with `imx8mm-lpddr4-evk` and the firmware is installed on the image
```
# find tmp-lmp/work/imx8mm_lpddr4_evk-lmp-linux/lmp-base-console-image/1.0-r0/rootfs/| grep "firmware/nxp"
tmp-lmp/work/imx8mm_lpddr4_evk-lmp-linux/lmp-base-console-image/1.0-r0/rootfs/usr/lib/firmware/nxp
tmp-lmp/work/imx8mm_lpddr4_evk-lmp-linux/lmp-base-console-image/1.0-r0/rootfs/usr/lib/firmware/nxp/txpwrlimit_cfg_8987.conf
tmp-lmp/work/imx8mm_lpddr4_evk-lmp-linux/lmp-base-console-image/1.0-r0/rootfs/usr/lib/firmware/nxp/ed_mac_ctrl_V3_8987.conf
tmp-lmp/work/imx8mm_lpddr4_evk-lmp-linux/lmp-base-console-image/1.0-r0/rootfs/usr/lib/firmware/nxp/sdiouart8987_combo_v0.bin
tmp-lmp/work/imx8mm_lpddr4_evk-lmp-linux/lmp-base-console-image/1.0-r0/rootfs/usr/lib/firmware/nxp/wifi_mod_para.conf
```

and the correspondent new packages 
```
# grep firmware-nxp-wifi build-imx8mm-lpddr4-evk/deploy/images/imx8mm-lpddr4-evk/lmp-base-console-image-imx8mm-lpddr4-evk.manifest
firmware-nxp-wifi-bcm4359-pcie all 1.0-r0
firmware-nxp-wifi-nxp-common all 1.0-r0
firmware-nxp-wifi-nxp8987-sdio all 1.0-r0

```

**Pull request merged**